### PR TITLE
Refactor sign up button

### PR DESCRIPTION
**Pull request by cortezroberto**
We've refactored the form button to fix CodeClimate issues.

Co-authored-by: yovanha <yfajardo0@ucol.mx>
Co-authored-by: hancel <hancel@gmail.com>

# Description

 - We've refactored the sign up button.
 - We fixed CodeClimate issues duplicated code.

### CODE REVIEW

File: FlightReservation/src/components/form/ButtonForm.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import style from '../../consts/style';
+
+
+const ButtonForm = ({isValid, handleSubmit, loading, text}) => {
+    return (
+        <Pressable
+            disabled={!isValid || loading}
+            onPress={handleSubmit}>
+            <View style={style.btn}>
+                <Text style={{color: colors.white, fontSize: 18}}>
```
Comment (DemisRincon) [inline FlightReservation/src/components/form/ButtonForm.js:14]: Please avoid inline styles

File: FlightReservation/src/screens/SignUp.js [hunk 2]
```diff
@@ -82,33 +83,15 @@ const SignUp = function ({navigation}) {
...
-                    </Text>
-                  </View>
-                </Pressable>
-
+                <ButtonForm isValid = {formProps.isValid} handleSubmit = {formProps.handleSubmit} text = "Sign up" loading = {loading}/>
                 <Text
                   style={[
                     style.lower_content_text,
                     {paddingTop: 10, paddingBottom: 10},
                   ]}>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/SignUp.js:91]: avoid inline styles

File: FlightReservation/src/screens/SignUp.js [hunk 3]
```diff
@@ -82,33 +83,15 @@ const SignUp = function ({navigation}) {
...
-                  onPress={formProps.handleSubmit}>
-                  <View style={style.btn}>
-                    <Text style={{color: colors.white, fontSize: 18}}>
-                      Sign Up with Google
-                    </Text>
-                  </View>
-                </Pressable>
+                <ButtonForm isValid = {formProps.isValid} handleSubmit = {formProps.handleSubmit} text = "Sign Up with Google" loading = {loading}/>
                 <View style={style.lower_content_text}>
                   <Text style={{paddingTop: 20}}>
```
Comment (DemisRincon) [inline FlightReservation/src/screens/SignUp.js:96]: avoid inline styles

**Review by DemisRincon — CHANGES_REQUESTED** [review]
Please avoid the use of the inline styles, put those into a stylesheet

**Review by YilKanda — COMMENTED** [review]
Just as an addition, try to use Internalization instead of static text.

You can use this library, for example, to do this:

[react-native-i18n](https://www.npmjs.com/package/react-native-i18n)

File: FlightReservation/src/components/form/ButtonForm.js (hunk 1 above)
Comment (cortezroberto) [reply FlightReservation/src/components/form/ButtonForm.js]: We've fixed all the inline styles in the Sign up form and ButtonForm component.

File: FlightReservation/src/screens/SignUp.js (hunk 2 above)
Comment (cortezroberto) [reply FlightReservation/src/screens/SignUp.js]: We've fixed all the inline styles in the Sign up form and ButtonForm component.

**Review by hancel-avila — APPROVED** [review]
Nice work with the style import

**Review by DemisRincon — APPROVED** [review]
styles and refactoring done, good

**Review by DiegoSpielzeug — APPROVED** [review]
LGTM

**Review by jkzasori — APPROVED** [review]

**Pull request merged**